src/main/java/in/ac/bits/protocolanalyzer/utils/ByteOperator.java

```diff
@@ -1,7 +1,22 @@
 package in.ac.bits.protocolanalyzer.utils;
 
+/**
+ * This utility allows for various byte operations.
```

Comment:
The class contains utility methods for converting byte range into chosen primitive data types (int, short, long etc.).